Enable verbose mode when building.

Don't regenerate rdoc documentation during install.

Index: common.mk
--- common.mk.orig
+++ common.mk
@@ -9,7 +9,7 @@ dll: $(LIBRUBY_SO)
 .SUFFIXES: .rbinc .rb .inc .h .c .y .i .$(ASMEXT) .$(DTRACE_EXT)
 
 # V=0 quiet, V=1 verbose.  other values don't work.
-V = 0
+V = 1
 V0 = $(V:0=)
 Q1 = $(V:1=)
 Q = $(Q1:0=@)
@@ -639,7 +639,7 @@ dont-install-man: $(PREP)
 post-no-install-man::
 	@$(NULLCMD)
 
-install-doc: rdoc pre-install-doc do-install-doc post-install-doc
+install-doc: pre-install-doc do-install-doc post-install-doc
 pre-install-doc:: install-prereq
 do-install-doc: $(PROGRAM) pre-install-doc
 	$(INSTRUBY) --make="$(MAKE)" $(INSTRUBY_ARGS) --install=rdoc $(INSTALL_DOC_OPTS)
